src/main/java/core/basesyntax/Solution.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Solution {
+    public static void main(String[] args) {
+        Pair<Integer, String> pair = Pair.of(1, "hello");
+        Integer i = pair.getFirst(); // 1
```

Comment:
since we don't need this class in solution next time don't add it to commit, ok ?